torch/_inductor/codegen/triton.py

```diff
@@ -1295,7 +1304,7 @@ def bucketize(
         self,
         values: CSEVariable,
         offsets_name: str,
-        offsets_size,
+        offsets_size: sympy.Expr,
```

Comment:
@aakhundov suggested we just pass in a single `offsets` TensorBox here - I didn't do it because it seems like it would require a bit of refactoring, but if others think that's a better option, LMK and I can do that